core/commands/bitswap.go

```diff
@@ -73,43 +78,43 @@ var unwantCmd = &cmds.Command{
 	},
 }
 
-var showWantlistCmd = &cmds.Command{
-	Helptext: cmds.HelpText{
+var showWantlistCmd = &oldcmds.Command{
+	Helptext: cmdkit.HelpText{
 		Tagline: "Show blocks currently on the wantlist.",
 		ShortDescription: `
 Print out all blocks currently on the bitswap wantlist for the local peer.`,
 	},
-	Options: []cmds.Option{
-		cmds.StringOption("peer", "p", "Specify which peer to show wantlist for. Default: self."),
+	Options: []cmdkit.Option{
+		cmdkit.StringOption("peer", "p", "Specify which peer to show wantlist for. Default: self."),
 	},
 	Type: KeyList{},
-	Run: func(req cmds.Request, res cmds.Response) {
+	Run: func(req oldcmds.Request, res oldcmds.Response) {
 		nd, err := req.InvocContext().GetNode()
 		if err != nil {
-			res.SetError(err, cmds.ErrNormal)
+			res.SetError(err, cmdkit.ErrNormal)
 			return
 		}
 
 		if !nd.OnlineMode() {
-			res.SetError(errNotOnline, cmds.ErrClient)
+			res.SetError(errNotOnline, cmdkit.ErrClient)
 			return
 		}
 
 		bs, ok := nd.Exchange.(*bitswap.Bitswap)
 		if !ok {
-			res.SetError(u.ErrCast(), cmds.ErrNormal)
+			res.SetError(e.TypeErr(bs, nd.Exchange), cmdkit.ErrNormal)
```

Comment:
clever, since typeof(bs) will be whatever we were trying to cast it to, right?

Reply:
exactly